module/importer/src/Infrastructure/Handler/ImportOptionCommandHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Importer\Infrastructure\Handler;
+
+use Ergonode\Importer\Infrastructure\Exception\ImportException;
+use Ergonode\Importer\Domain\Entity\ImportError;
+use Ergonode\Importer\Domain\Repository\ImportErrorRepositoryInterface;
+use Ergonode\Importer\Infrastructure\Action\OptionImportAction;
+use Ergonode\Importer\Domain\Command\Import\ImportOptionCommand;
+
+/**
+ */
+class ImportOptionCommandHandler
+{
+    /**
+     * @var OptionImportAction
+     */
+    private OptionImportAction $action;
+
+    /**
+     * @var ImportErrorRepositoryInterface
+     */
+    private ImportErrorRepositoryInterface $repository;
+
+    /**
+     * @param OptionImportAction             $action
+     * @param ImportErrorRepositoryInterface $repository
+     */
+    public function __construct(OptionImportAction $action, ImportErrorRepositoryInterface $repository)
+    {
+        $this->action = $action;
+        $this->repository = $repository;
+    }
+
+    /**
+     * @param ImportOptionCommand $command
+     *
+     * @throws \Exception
+     */
+    public function __invoke(ImportOptionCommand $command)
+    {
+        try {
+            $this->action->action(
+                $command->getCode(),
+                $command->getKey(),
+                $command->getTranslation()
+            );
+        } catch (ImportException $exception) {
```

Comment:
Theese 3 line is copy/pasted to every import command handler. Maybe better option will be decorator or abstract or event trait (like in doctrine)